src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -160,6 +190,61 @@ class ActivityFeed extends React.Component<Props, State> {
         return feedItems.length === 0 || (feedItems.length === 1 && feedItems[0].type === ItemTypes.fileVersion);
     };
 
+    getActiveEntry = (callback: (activeEntry: FeedItem | null, repliesNeedToBeLoaded?: boolean) => void) => {
+        const { activeFeedEntryId, activeFeedEntryType, feedItems, getComment, hasReplies } = this.props;
+
+        if (!activeFeedEntryId || !Array.isArray(feedItems)) {
+            return callback(null);
+        }
+
+        const firstLevelItem = feedItems.find(
+            ({ id, type }) => id === activeFeedEntryId && type === activeFeedEntryType,
+        );
+        // If the active entry is a first level Feed item, return it
+        if (firstLevelItem) {
+            return callback(firstLevelItem);
+        }
+
+        // Check if hasReplies is enabled and active entry's type is comment
+        if (!hasReplies || activeFeedEntryType !== FEED_ITEM_TYPE_COMMENT) {
+            return callback(null);
+        }
+
+        const firstLevelItemWithActiveReply = feedItems.find(item => {
+            if (
+                (item.type !== FEED_ITEM_TYPE_ANNOTATION && item.type !== FEED_ITEM_TYPE_COMMENT) ||
+                !item.replies ||
+                !Array.isArray(item.replies)
+            ) {
+                return false;
+            }
+            return item.replies.some(({ id }) => id === activeFeedEntryId);
+        });
+        // If the active entry is within present replies of any first level Feed items (if it's a comment or annotation),
+        // return that first level Feed item
+        if (firstLevelItemWithActiveReply) {
+            return callback(firstLevelItemWithActiveReply);
+        }
+
+        if (!getComment) {
```

Comment:
Is this possible with the current implementation? It looks like the method is always defined.